.github/ISSUE_TEMPLATE/ask_question.md

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
TIP: I would convert all these `ISSUE_TEMPLATE`s to issue forms  and add them in the cookiecutter. It simplifies interaction with users and in the near future we will add support links to the metadata.

https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms